### PR TITLE
Authtests

### PR DESCRIPTION
**Pull request by infobiac**

### CODE REVIEW

File: tests/test_authentication_views.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        access_token="Bye", institution_name="bankofcool",
+        current_balance_field=10, account_name_field="coolaccount",
+        income_field=30, expenditure_field=5
+    )
+    userbank.save()
+    set_up = test_client.get("/setup_bank")
+    response_code_2 = set_up.status_code
+    assert response_code_2 == 302
+    url = set_up.url
+    assert url == "/home"
```
Comment (Neitsch) [inline tests/test_authentication_views.py:45]: Do you need to mock plaid for this testcase?

File: tests/test_stocks_model.py [hunk 2]
```diff
@@ -352,3 +352,40 @@ def test_bucket_value_on():
...
+@pytest.mark.django_db(transaction=True)
+def test_bucket_historical():
+    """
+    Tests InvestmentBucket.historical()
+    """
+    user = User.objects.create(username='user1', password="a")
+    stock = Stock(
+        name="Name1X",
+        ticker="TKRC"
+    )
```
Comment (Neitsch) [inline tests/test_stocks_model.py]: This will trigger a signal to fetch stock quotes. Disconnect it.

**Review by Neitsch — CHANGES_REQUESTED** [review]
Overall good

File: tests/test_authentication_views.py (hunk 1 above)
Comment (infobiac) [reply tests/test_authentication_views.py:45]: I don't think so? What I'm testing is just to see how the view reacts to when we have a userbank setup vs. when we don't and I set up the userbank manually. It shouldn't interact with/trigger plaid at all.

File: tests/test_stocks_model.py (hunk 2 above)
Comment (infobiac) [reply tests/test_stocks_model.py]: Good call, thanks.

Comment (infobiac) [reply tests/test_stocks_model.py]: wait isn't it disconnected by the setup/teardown already?

**Review by Neitsch — APPROVED** [review]

**Pull request merged**